include/bluetooth/services/fast_pair.h

```diff
@@ -36,15 +36,20 @@ size_t bt_fast_pair_adv_data_size(bool fp_discoverable);
  *
  * The buffer size must be at least @ref bt_fast_pair_adv_data_size.
  *
- * @param[out] adv_data		Pointer to the Bluetooth advertising data structure to be filled.
- * @param[out] buf		Pointer to the buffer used to store Fast Pair advertising data.
- * @param[in]  buf_size		Size of the buffer used to store Fast Pair advertising data.
- * @param[in]  fp_discoverable	Boolean indicating if device is Fast Pair discoverable.
+ * @param[out] adv_data		     Pointer to the Bluetooth advertising data structure to be
+ *                                   filled.
+ * @param[out] buf		     Pointer to the buffer used to store Fast Pair advertising data.
+ * @param[in]  buf_size		     Size of the buffer used to store Fast Pair advertising data.
+ * @param[in]  fp_discoverable	     Boolean indicating if device is Fast Pair discoverable.
+ * @param[in]  fp_show_ui_indication Boolean indicating if UI indication are to be shown on Fast
+ *                                   Pair Seeker device. This parameter is used only, if device is
+ *                                   not Fast Pair discoverable and the Account Key List is not
+ *                                   empty.
  *
  * @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
  */
 int bt_fast_pair_adv_data_fill(struct bt_data *adv_data, uint8_t *buf, size_t buf_size,
-			       bool fp_discoverable);
+			       bool fp_discoverable, bool fp_show_ui_indication);
```

Comment:
I would add a separate API to toggle this, because I would prefer to keep it as a separate state. Something similar to bt_fast_pair_set_pairing_mode. Then our approach would also be more consistent in both cases.

Reply:
Current approach also introduces a parameter which is ignored in some cases, I would prefer to avoid it.